lib/service/Service.ts

```diff
@@ -545,6 +572,24 @@ class Service {
     argChecks.POSITIVE_AMOUNT(request);
     return this.swaps.handleResolveRequest(request);
   }
+  /**
+   * Resolves alias to node's public key
+   */
+  private resolveAlias = async (alias: string) => {
+    const plist = this.listPeers();
+    const keys: string[] = [];
+    for (const peer of plist) {
+      if (peer.alias) {
+        if (peer.alias.toLowerCase() === alias.toLowerCase()) {
```

Comment:
it's just so that the user does not have to worry about capitalization when using the alias, so it's a UX thing

Reply:
But the user is never inputting aliases, right? I just don't see how the casing could become inconsistent such that it would affect this conditional.

Reply:
Not as of now, but in future. The idea is that you can use an alias instead of a node key for commands like e.g. `openchannel`. So I think `.toLowerCase()` is a good idea.

Reply:
no this PR allows the user to input aliases which are functionally interchangeable with the public key. Basically the user is likely to input "partapartb" instead of "PartAPartB" and it's a better feel at the command line if either works